code/modules/reagents/reagent_containers/chemical_bottle.dm

```diff
@@ -12,6 +12,10 @@
 	container_type = OPENCONTAINER
 	volume = 30
 
+/obj/item/reagent_containers/glass/bottle/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>[src] can hold up to [reagents.maximum_volume] units.</span>"
```

Comment:
This will result in a double examine, as the examine from `/obj/item/reagent_containers/glass` is also run!

Reply:
oh ok, so this is not needed?